src/Schema/Keywords/UniqueItems.php

```diff
@@ -39,7 +42,14 @@ public function validate($data, bool $uniqueItems) : void
             throw InvalidSchema::becauseDefensiveSchemaValidationFailed($e);
         }
 
-        if (count(array_unique($data)) !== count($data)) {
+        $items = $data;
+        if (is_array($data) && count($data)) {
```

Comment:
I don't really thing that we need both checks here. It's known it is array data, because of assertion above. And if we won't check count than we will just get empty array if it's initially empty. So we can omit this check entirely and get more simple code